src/payment/strategies/square/square-form.ts

```diff
@@ -39,7 +92,8 @@ export interface SquareFormElement {
 export interface SquareFormCallbacks {
     paymentFormLoaded?(form: SquarePaymentForm): void;
     unsupportedBrowserDetected?(): void;
-    cardNonceResponseReceived?(errors: any, nonce: string): void;
+    cardNonceResponseReceived?(errors: NonceGenerationError[] | null, nonce: string, cardData: CardData, billingContact: Contact | undefined,
```

Comment:
There's seems to be a mismatch between the types here and the types https://github.com/bigcommerce/checkout-sdk-js/pull/371/files#diff-176440f483d3bb25eec518d2c5c27389R134

Reply:
👍